### ISSUE TITLE
on Ubuntu 22.04 bindkeys -e must be set before loading this module for <s-tab> to work

### ISSUE THREAD

**Issue by Konfekt** [zimfw/input#7]
### Prerequisites

- [X] I've checked the [existing issues](../issues?q=is%3Aissue) and I'm not duplicating a report.
- [X] I've checked the zimfw [Changelog](https://github.com/zimfw/zimfw/blob/master/CHANGELOG.md) and I'm not being affected by documented changes.
- [X] I've checked the zimfw [📢 Announcements](https://github.com/zimfw/zimfw/discussions/426?sort=new) and I'm not being affected by announced changes.

### Bug description

On Ubuntu 22.04 bindkeys -e must be set before loading this module for <s-tab> to work;
otherwise `<s-tab>` does nothing;
it can be enabled with `bindkey '^[[Z' reverse-menu-complete` but this module usually does this.
It did not happen, say, on Opensuse 15.5

### Steps to reproduce

On Ubuntu 22.04 set bindkeys -e after loading this module for <s-tab> to break

### Current behavior

On Ubuntu 22.04 set bindkeys -e before loading this module makes <s-tab> work

### Expected behavior

On Ubuntu 22.04 set bindkeys -e after loading this module lets <s-tab> work as on Opensuse 15.5

### zimfw info

```shell
zimfw version:        1.13.1 (built at 2024-04-28 19:07:19 UTC, previous commit is c86223f)
LANG:                 it_IT.UTF-8
LC_TELEPHONE:         it_IT.UTF-8
LC_MEASUREMENT:       it_IT.UTF-8
LC_ADDRESS:           it_IT.UTF-8
LC_COLLATE:           C
LC_NAME:              it_IT.UTF-8
LC_NUMERIC:           it_IT.UTF-8
LC_IDENTIFICATION:    it_IT.UTF-8
LC_PAPER:             it_IT.UTF-8
LC_MONETARY:          it_IT.UTF-8
LC_TIME:              it_IT.UTF-8
LC_MESSAGES:          it_IT.UTF-8
LC_CTYPE:             it_IT.UTF-8
OSTYPE:               linux-gnu
TERM:                 rxvt-unicode-256color
TERM_PROGRAM:         
TERM_PROGRAM_VERSION: 
ZIM_HOME:             /home/konfekt/.config/.zim
ZSH_VERSION:          5.8.1
```

**Comment by ericbn** [comment]
Ciao @Konfekt. The zshrc template in our automatic installation script does call bindkeys before initializing the modules. See https://github.com/zimfw/install/blob/30c5707a2e613aa388846caa9b2aee4ac20d6059/src/templates/zshrc#L22

Do you suggest we also mention this in this module’s README?

**Comment by Konfekt** [comment]
Yes, I somehow missed this, and I wondered why what worked fine on one Linux distrib no longer worked on another one

**Comment by Konfekt** [comment]
I would appreciate an idea how it could be that it worked in the first place, even if the set-up wasn't what it was meant to be

**Comment by Konfekt** [comment]
.. Well.. I think the solution is that the former distrib sets bindkey in `/etc`; for once that went well
